src/kani-compiler/cbmc/src/goto_program/expr.rs

```diff
@@ -851,13 +852,10 @@ impl Expr {
             // Floating Point Equalities
             IeeeFloatEqual | IeeeFloatNotequal => lhs.typ == rhs.typ && lhs.typ.is_floating_point(),
             // Overflow flags
-            OverflowMinus => {
-                (lhs.typ == rhs.typ && (lhs.typ.is_pointer() || lhs.typ.is_numeric()))
```

Comment:
Good to restrict this case to integers because I do not think we were handling floating point numbers anyway.